ironfish/src/network/messages/gossipNetworkMessage.ts

```diff
@@ -13,7 +13,7 @@ export abstract class GossipNetworkMessage extends NetworkMessage {
   constructor(type: NetworkMessageType, nonce?: Buffer) {
     super(type)
 
-    this.nonce = nonce ?? Buffer.from(tweetnacl.randomBytes(16))
+    this.nonce = nonce ?? Buffer.from(randomBytes(16))
```

Comment:
I don't think this has to change, but this could probably just use `import { randomBytes } from 'crypto'` to reduce the API surface of the Rust code a bit -- tweetnacl was just calling that internally anyway.

Reply:
Ok but hear me out. This is 1.5 - 2x faster than javascript's crypto randomBytes

(Granted, we're talking about 2-4 microseconds vs 3-7 microseconds, so y'know, world's greatest over-optimzation)